statistics/selectivity.go

```diff
@@ -53,6 +54,27 @@ const (
 	ColType
 )
 
+func compareType(l, r int) int {
+	if l == r {
+		return 0
+	}
+	if l == ColType {
+		return -1
+	}
+	if l == PkType {
+		return 1
+	}
+	if r == ColType {
+		return 1
+	}
+	return -1
```

Comment:
It seems we can modify the `iota` order and directly compare it with `<`.

Reply:
> In order to keep compatibility, I introduces the compareType function, instead of changing the values of IndexType / PkType / ColType, because feedback encoding uses these constants.

Reply:
Okay.